appveyor.yml

```diff
@@ -13,17 +20,18 @@ platform:
 shallow_clone: true
 
 install:
-  - ps: Install-Product node $env:nodejs_version $env:Platform
+  - ps: Install-Product node $env:nodejs_version $env:platform
   - ps: Set-ExecutionPolicy Unrestricted -Scope CurrentUser -Force
   - npm config get
   - node --version
   - npm --version
-  - node -e "console.log(process.arch);"
-  - IF /I "%PLATFORM%" == "x64" set PATH=C:\Python27-x64;%PATH%
-  - IF /I "%PLATFORM%" == "x86" SET PATH=C:\python27;%PATH%
-  - IF /I "%PLATFORM%" == "x64" CALL "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" amd64
-  - IF /I "%PLATFORM%" == "x86" CALL "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" x86
+  - node -e "console.log(process.arch);"  # 'x64' or 'ia32'
+  # Python >= v3.12 requires node-gyp >= v10
+  - IF /I "%PLATFORM%" == "x64" set PATH=C:\Python311-x64;%PATH%
+  - IF /I "%PLATFORM%" == "x86" SET PATH=C:\python311;%PATH%
+  - npm config list
   - npm ci
```

Comment:
```suggestion
  - npm ci
  - npm run update-crosswalk  # Support newer versions of Node.js
```

Reply:
this isn't going to be able to write back to the repo, so what is the point of running npm run update-crosswalk here, just to see if it works?

Reply:
To document what needs to be done to MAKE it work.